### PR TITLE
Media Query Addition for Desktop version

### PR DESCRIPTION
**Pull request by Addisu87**
I have installed the GitFlow and merged two branches, added_sections were in GithubFlow and "mediaquery" branch with GitFlow. I have merged those into one added_sections branch. Moreover, I have added the media query width from 768px to 1440px and made it responsive.

### CODE REVIEW

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @Addisu87 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no pull request description
image

Kindly add a description and someone will review your work.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Addisu87** [comment]
Changing different png into SVG file and styling as the Figma design.

**Review by youmari — CHANGES_REQUESTED** [review]
#
Hi @Addisu87 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] Doesn't have a Descriptive short summary 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Addisu87** [comment]
@[youmari](https://github.com/youmari) where can I add this short summary.

File: style.css [hunk 1]
```diff
@@ -314,11 +428,144 @@ input:focus:invalid {
...
+.shape {
+  width: 135px;
+  display: center;
+  margin: 32px 8px 12px 22px;
+}
+
+/* Media Queries section */
+
+@media screen and (min-width: 1440px) {
+  /* navbar */
```
Comment (arslanbisharat) [inline style.css:442]: - [ ] Use media queries for the Desktop breakpoint in the CSS file. (the value for `min-width` property should be in range of `768px` or wider)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure you are using GitFlow. Currently, you are using github flow. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by youmari** [comment]
> @[youmari](https://github.com/youmari) where can I add this short summary.

you can add it at the top of the page  click on the three-dot then edit to add it
image

**Comment by Addisu87** [comment]
@[youmari](https://github.com/youmari) I have managed to add GitFlow but the branches are diverted. I think the first one is in GitHub flow and the other is using GitFlow. Shall I create another PR or simply request to use the new one. If you have any suggestions please send me.

**Review by RolandM99 — APPROVED** [review]
<h1>Status: Approve 🥇 ✔</h1>
Hi @Addisu87 , 

### Now your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Now you can take a rest a prepare your next week!
## Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**